README.md

```diff
@@ -113,13 +113,13 @@ GitHub Pages
 
```

Comment:
- [ ] As suggested by the previous reviewer you have submitted a project, but the GitHub flow is incorrect.
 Kindly open a PR from a feature branch into the main branch. Consider this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) for further information.